### PR TITLE
refactor: Remove experimental config for appDir and @next/font

### PR DESCRIPTION
**Pull request by grzegorzpokorski**
Changes:
- Removed experimental config for appDir because starting from nextjs v13.4, this feature is stable (for now in the console at development mode we have some warns about that this config is necessary but it was fixed in the latest version of nextjs),
- Removed the `@next/font` package since it is now built into nextjs itself.

### CODE REVIEW

File: examples/example-next-13-next-auth/next.config.js [hunk 1]
```diff
@@ -1,3 +1 @@
-module.exports = {
-  experimental: {appDir: true}
-};
+module.exports = {};
```
Comment (amannn) [inline examples/example-next-13-next-auth/next.config.js:1]: Maybe we can remove the config files entirely now if they're empty?

File: examples/example-next-13/next.config.js [hunk 2]
```diff
@@ -1,3 +1 @@
-module.exports = {
-  experimental: {appDir: true}
-};
+module.exports = {};
```
Comment (amannn) [inline examples/example-next-13/next.config.js:1]: Same here.

**Review by amannn — COMMENTED** [review]
Many thanks for this PR, always great to see old cruft removed! 🙌

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -1,4 +1,8 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```
Comment (amannn) [inline pnpm-lock.yaml:1]: I think the pnpm version needs to be updated in the GitHub workflows for this to work.

File: examples/example-next-13-next-auth/next.config.js (hunk 1 above)
Comment (grzegorzpokorski) [reply examples/example-next-13-next-auth/next.config.js:1]: Good point, i didn't thought about it 😂

**Review by amannn — APPROVED** [review]
Many thanks! 🙌

**Pull request merged**